cardano-api/src/Cardano/Api/Script.hs

```diff
@@ -879,6 +894,24 @@ data SimpleScript lang where
 deriving instance Eq   (SimpleScript lang)
 deriving instance Show (SimpleScript lang)
 
+instance Ord (SimpleScript lang) where
```

Comment:
I'm extremely sceptical that we need to order actual scripts by their value. At most I can imagine a round trip via the ledger types putting things into order based on hash, but not the value. This must be unnecessary, surely?